static/js/util/ProfileFormFields.js

```diff
@@ -1,3 +1,4 @@
+// @flow
```

Comment:
Is there another way to specify which files we want to cover? I'm fine with this for now but if we end up running this on most JS files we have, it would be nice to use a exclude instead if that's supported by flow